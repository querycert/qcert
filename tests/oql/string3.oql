select struct(name:x->name)
from x as Customer in WORLD
