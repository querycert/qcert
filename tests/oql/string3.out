[{"name" : "John Doe"},
 {"name" : "Jane Doe"},
 {"name" : "Jim Does"},
 {"name" : "Jill Does"},
 {"name" : "Joan Doe"},
 {"name" : "James Do"}]